### PR TITLE
[DOC-7128] Saved views

### PR DESCRIPTION
**Pull request by urbiz-nr**
- Added saved views in Explorer doc
- Updated a couple of screenshots

See DOC-7128 for more info.

### CODE REVIEW

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:71]: style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:79]: style guide uses Oxford comma so s/be "navigate, group, and filter"

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:83]: s/be `OR` with the backticks

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:84]: I'm wondering if this long sentence could be broken like this? "...for exploratory purposes. It also allows you to see only the entities matching the `type` selected. You can also update the filter from the filter bar." I'm not sure I reworded this to keep what you intended to convey, sorry!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:87]: This new section is excellent! Really helpful info, congrats!

File: src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx
Comment (barbnewrelic) [inline src/content/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one.mdx:24]: style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
I added a few questions and comments inline. After you have a chance to look them over, feel free to use your admin powers to merge directly. No need for a second review request from me. Really nice, congrats!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:71]: Done! I always forget this!

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:84]: I've made a couple of tweaks.

File: src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx
Comment (urbiz-nr) [reply src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx:87]: Thanks!

**Review by x8a — APPROVED** [review]
👏🏻

**Pull request merged**
